src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -139,18 +111,33 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
         (accountId ?? highestBalanceAccountId ?? defaultAccountId)!,
         toOpportunityId({
           chainId,
-          assetNamespace: 'erc20',
-          assetReference: contractAddress,
+          assetNamespace,
+          assetReference,
         }),
       ),
     }),
-    [accountId, chainId, contractAddress, defaultAccountId, highestBalanceAccountId],
+    [accountId, assetNamespace, assetReference, chainId, defaultAccountId, highestBalanceAccountId],
   )
 
   const opportunityData = useAppSelector(state =>
     selectEarnUserStakingOpportunityByUserStakingId(state, opportunityDataFilter),
   )
 
+  const underlyingAssetsFiatBalanceCryptoPrecision = useMemo(() => {
+    if (!asset || !opportunityData?.underlyingAssetId) return '0'
+
+    const cryptoAmount = bnOrZero(opportunityData?.stakedAmountCryptoBaseUnit)
+      .div(bn(10).pow(asset.precision))
+      .toString()
```

Comment:
```suggestion
      .toFixed()
```

not sure if we want fixed here or string?

Reply:
Copied from `FoxFarmingManager`, if that's an issue, let's update all occurrences in a follow-up PR @0xdef1cafe ?
Given all we're doing here is using this amount to calculate a rough fiat amount, this is safe IMO